datalad/distribution/dataset.py

```diff
@@ -384,16 +391,20 @@ def get_superdataset(self, topmost=False):
             # None was found
             return None
 
-        if realpath(self.path) != self.path:
-            # we had symlinks in the path but sds_path would have not
-            # so let's get "symlinked" version of the superdataset path
-            sds_relpath = relpath(sds_path, realpath(self.path))
-            sds_path = normpath(opj(self.path, sds_relpath))
+        # I think we might have accounted for it with use of normpath there??
```

Comment:
Well, `normpath` would remove the trailing `/..`, yes. But "accounting for symlinks" is a misleading wording, I think, since `normpath` doesn't care for symlinks at all. It just normalizes the path to the canonical form.